mne/preprocessing/maxwell.py

```diff
@@ -449,6 +449,7 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
         raw_sss._data[pos_picks, start:stop] = out_pos_data
 
     # Update info
+    info['dev_head_t'] = recon_trans  # set the reconstruction transform
```

Comment:
Also, was there a reason for not folding this change into the `_update_sss_info` function?